app/src/androidTest/java/com/epfl/neighborfood/neighborfoodandroid/ui/activities/SignUpActivityTest.java

```diff
@@ -32,30 +37,39 @@
 public class SignUpActivityTest {
 
     // a fake LoggedInUser that will be used to simulate the case where an actual account is logged in into the app
-    private User fakeUser = new UserTestImplementation("0", "FakeEmail@gmail.com", "Firstname", "Lastname");
+    private AuthenticatorUser fakeUser = new AuthenticatorUser("0", "FakeEmail@gmail.com", "Firstname", "Lastname", "") {
+    };
 
     @Rule
     public ActivityScenarioRule<SignUpActivity> testRule = new ActivityScenarioRule<>(SignUpActivity.class);
 
+    @BeforeClass
+    public static void setupApp(){
+        NeighborFoodApplication.appContainer = new AppContainerTestImplementation();
+        //NeighborFoodApplication.appContainer.getAuthRepo().logInWithGoogleAccount(null);
```

Comment:
Should not leave commenter lines I think

Reply:
I got in too deep doing the modifications and realsied too late that that's what I should have done, this is noted and taken into consideration